backend/test_suite_generator.py

```
@@ -13,7 +13,8 @@
 import string
 from datetime import datetime, timezone, timedelta
 from typing import Dict, Any, Union
-import logging
+import logging 
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)